pkg/chart/chart_test.go

```diff
@@ -31,17 +29,11 @@ import (
 type fakeGit struct{}
 
 func (g fakeGit) FileExistsOnBranch(file string, remote string, branch string) bool {
-	_, err := os.Open(computePreviousRevisionPath(file))
-	fmt.Println(err)
-	return err == nil
+	return true
 }
 
 func (g fakeGit) Show(file string, remote string, branch string) (string, error) {
-	b, err := ioutil.ReadFile(computePreviousRevisionPath(file))
-	if err != nil {
-		return "", err
-	}
-	return string(b), nil
```

Comment:
This stuff was no longer necessary after https://github.com/helm/chart-testing/pull/120/commits/9616010a874d8b6bf698a2cb6e36381848119044.